docs/source/user_guide/app.rst

```diff
@@ -1056,6 +1056,16 @@ You can also programmatically configure a session's color scheme by creating
     Refer to the |ColorScheme| class for documentation of the available
     customization options.
 
+.. note::
+
+    For maskTargetsColors, intTarget must be an integer between 1 and 255. 
+    Do not set color for intTarget 0, as it is reserved. 
+
+.. note::
+
+    For colorscales, the list of colors must be sorted by value in ascending
```

Comment:
Can we instead update the frontend so it sorts by `value`? Should be a simple improvement right?

Reply:
The FE already does it, but I can put the sort in the get_colormap code. And will add more validation to the class to let users know if input is not valid.

Reply:
Okay great, definite +1 to adding the sorting logic to `get_colormap()`.

Reply:
Just tested with all types of inputs. Plotly now already handle unsorted list and more than rgb colors. We don't need to change any code. I will remove this note from documentation.